src/abstractions/serialization/KiotaJsonSerializer.Deserialization.Untyped.cs

```diff
@@ -0,0 +1,88 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+
+#if NET5_0_OR_GREATER
+
+using System.Threading.Tasks;
+using System.Threading;
+using System.IO;
+using System;
+using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections.Generic;
+using System.Linq;
+
+#pragma warning disable IL3050 
```

Comment:
can you provide more context around this suppression please?

Reply:
Well the Warning I received was, that AOT will not have the type ready when using a native assembly and will have to create it during runtime. At least that is, what I got from the message and some reading on the internet.
 Nevertheless, this is intended behavior, because this is exactly for the case, when one does not know the Type at compile time and therefore needs this API in the first place. Otherwise one could have used the Generic Methods instead of these.